ChangeLog

```diff
@@ -1,3 +1,6 @@
+22.3.4
+ - Fix Oracle DS primary interface when using IMDS (#1757)
```

Comment:
Let's annotate this documenting the original bug too
```suggestion
 - Fix Oracle DS primary interface when using IMDS (LP: #1989686)
```

Then let's squash merge that change into https://github.com/canonical/cloud-init/pull/1761/commits/bcf94ba35d6e87186cef27b76b0724b061725657

Reply:
Done (tag updated as well)